### PR TITLE
Address old line-splitting TODO

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: lib/src/io.dart [hunk 1]
```diff
@@ -1240,8 +1239,12 @@ class PubProcessResult {
 
   // TODO(rnystrom): Remove this and change to returning one string.
```
Comment (sigurdm) [inline lib/src/io.dart:1240]: Is this TODO really resolved?

Comment (parlough) [reply lib/src/io.dart:1240]: No this one isn't, so I left it here. It would require a larger refactoring.

Comment (sigurdm) [reply lib/src/io.dart:1240]: Ah sorry, this PR is about the other TODO. Gotcha....

**Review by sigurdm — APPROVED** [review]
Thanks!

**Pull request merged**